ctapipe/instrument/trigger.py

```diff
@@ -0,0 +1,97 @@
+from ctapipe.containers import ArrayEventContainer
+from ctapipe.core import TelescopeComponent
+from ctapipe.core.traits import Integer, IntTelescopeParameter
+
+
+class SoftwareTrigger(TelescopeComponent):
+    """
+    A stereo trigger that can remove telescope events from subarray events.
+
+    This class is needed to correctly handle super-arrays as simulated for
+    CTA and still handle the LST hardware stereo trigger and the normal stereo
+    trigger correctly.
+
+    LSTs have a hardware trigger, that requires at least two LSTs to have a trigger
+    in each event. Thus, after selecting a subarray, it is needed to
+    - Remove single LST telescope events from events
+    - Ignore events with only 1 telescope after this has been applied
+    """
+
+    min_telescopes = Integer(
```

Comment:
I am not happy with the naming here, as only the help of the other parameter shows, that *after* applying other checks this one is used.  I was very confused when I read the tests (which I did before reading this file), that `min_telescopes` is (intentionally) lower than `sum(min_telescopes_of_type)`